front_end/content.py

```diff
@@ -48,6 +48,18 @@ def get_problems(course, assignment, show_hidden=True):
 
     return sort_nested_list(problems)
 
+def get_submissions(course, assignment, problem, user):
+    submissions = []
+
+    for submission_path in glob.glob(f"/submissions/{course}/{assignment}/{problem}/{user}/*"):
```

Comment:
To be consistent with the rest of the code, please create a function called get_submission_dir_path to gives you the path to this directory and use that rather than specifying it here. Make sure this new function is in the code after get_problem_dir_path.

Reply:
Done!